iOS/Flows/Conversation/Expressions/ExpressionViewController.swift

```diff
@@ -47,48 +49,49 @@ class ExpressionViewController: ViewController {
         
         self.view.addSubview(self.blurView)
                 
-        self.addChild(viewController: self.expressionPhotoVC)
+        self.addChild(viewController: self.expressionCaptureVC)
         
         self.view.addSubview(self.personGradientView)
         self.personGradientView.alpha = 0.0
+
+        self.view.addSubview(self.retakeButton)
+        self.retakeButton.set(style: .normal(color: .B1, text: "Retake"))
+        self.view.addSubview(self.doneButton)
+        self.doneButton.set(style: .normal(color: .B2, text: "Done"))
```

Comment:
For now I just made a Retake button instead of tapping the whole screen. This simplifies having to turn the long press gesture on and off.